examples/workflows/upstream/conditional_parameters.py

```diff
@@ -0,0 +1,60 @@
+from hera.workflows import Script, Step, Steps, Workflow
+from hera.workflows.models import Outputs, Parameter, ValueFrom
+
+with Workflow(
+    api_version="argoproj.io/v1alpha1",
+    kind="Workflow",
+    annotations={
+        "workflows.argoproj.io/description": "Conditional parameters provide a way to choose the output parameters based on expression.\n\nIn this example the step template has two steps which will run conditionally on `when`.\n\nBased on that condition, one of step will not be executed. The step template's output parameter will be\nset from the executed step's output.\n",
```

Comment:
Might want to split long strings somehow -- maybe at `\n`s -- so that the formatter can multiline them